defs/gmake.mk

```diff
@@ -357,6 +357,53 @@ ifneq ($(REVISION_IN_HEADER),$(REVISION_LATEST))
 $(srcdir)/revision.h: $(REVISION_H)
 endif
 
+# Show Cargo progress when doing `make V=1`
+CARGO_VERBOSE_0 = -q
+CARGO_VERBOSE_1 =
+CARGO_VERBOSE = $(CARGO_VERBOSE_$(V))
+
+.PHONY: yjit-static-lib
+yjit-static-lib:
+	$(Q) set -eu; if [ '$(YJIT_SUPPORT)' != 'dev' ]; then \
+	    echo 'building Rust YJIT (release mode)'; \
+	    $(RUSTC) \
+	        --crate-name=yjit \
+	        --crate-type=staticlib \
+	        --edition=2021 \
+	        -C opt-level=3 \
```

Comment:
Might not be in scope for this PR, but would we need to account for the cargo build target here when cross compiling ruby?

Since yjit is currently x86_64, I think these mappings should work:

| ruby | rust |
|------|------|
| x86_64-linux | x86_64-unknown-linux-gnu |
| x86_64-darwin | x86_64-apple-darwin |
| x64-mingw32 | x86_64-pc-windows-gnu |
| x64-mingw-ucrt | x86_64-pc-windows-gnu |